### PR TITLE
Delete tag related info

### PR DESCRIPTION
**Pull request by canyara**

### CODE REVIEW

File: net2/TagManager.js [hunk 1]
```diff
@@ -110,6 +110,22 @@ class TagManager {
         await rclient.delAsync(key);
         this.subscriber.publish("DiscoveryEvent", "Tags:Updated", null, this.tags[uid].o);
         await this.refreshTags();
+
```
Comment (jasonlyc) [inline net2/TagManager.js]: Better to move these logics into Tag.destroyEnv

Comment (canyara) [reply net2/TagManager.js]: The FireAPI process does not call Tag.destroyEnv

Comment (canyara) [reply net2/TagManager.js]: > Better to move these logics into Tag.destroyEnv

The FireAPI process does not call Tag.destroyEnv

Comment (jasonlyc) [reply net2/TagManager.js]: It can be done in the background

**Pull request merged**